WebDevStudios/wds-gutenberg
# Feature/#43 phase 1 release prep

Closes #43 

### DESCRIPTION ###
-  Remove install/update script
-  Remove ACF/MDB Pro/wd_s dependencies from readme
-  Rewrite readme to reflect updated development workflow
-  Add basic usage workflow to readme
-  Ensure branding and description is consistent
-  Ensure block names and icons are consistent with WDS branding
-  Wiki and documentation are on point

### SCREENSHOTS ###
N/A

### STEPS TO VERIFY ###
How do we test this?

Check out this PR and make sure everything still works and there are no remnants of "WDS Gutenberg"

### DOCUMENTATION ###
Will this pull request require updating the WDS Gutenberg [wiki](https://github.com/WebDevStudios/wds-gutenberg/wiki)?

No.
